scripts/populate_data.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
+
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+set -e
+
+# Pulls down NUM_CRASHES crashes per day for the last NUM_DAYS days, syncs the
+# crash data with local S3 container, and sends all crash ids to # the local
+# RabbitMQ queue for processing.
+#
+# Note: This assumes you've run "make dockersetup" and "make dockerupdatedata"
+# already.
+#
+# Usage:
+#
+#    $ ./docker/as_me.sh bash
+#    you@processor:/app$ ./scripts/populate_data.sh
+#
+# After running this script, run the processor:
+#
+#    $ docker-compose up processor
+#
+# Wait for crashes to process. Then you're good to go!
+
+NUM_DAYS=8
+NUM_CRASHES=15
+DATADIR=./crashdata_populate_tmp
```

Comment:
Do we want to use a temporary directory instead of hardcoding this?

Reply:
I hard-coded all the things above figuring it's a good start for now and we can hone it as issues come up.

If you want to write to write a function to generate a temporary directory, I'm game for using that.